### DISCUSSION TITLE
Advantages over official image

### DISCUSSION THREAD

**Discussion by madorin** [tiredofit/docker-collabora-online#42] [Q&A]
What are the advantages of using this image over official docker image?
Does your image support python scripting? are needed packages included?

**Comment by tiredofit** [comment]
The image was built to support some specific needs for a client..
- Back when it first started it allowed more connections simultaneously to documents and more concurrent users
- Ability to override things like fonts. Only recently have they introduced an API to support this
- Branding changes
- Monitoring/Metrics via Zabbix
- More configuration options

We could easily support Python either by the hooks or by introducing it baked into the image.

**Comment by aviatorm** [comment]
So, what's up for today?
Your docker image  allows more connections simultaneously to documents and more concurrent users (compared with original CODE)?

**Reply by tiredofit** [reply]
Still the same. Configuration options, in built monitoring, ability to compile the code (and perform modifications to it before hand automatically).

**Comment by aviatorm** [comment]
Thanks for the clarification.
